packages/devtools_app/lib/src/debugger/evaluate.dart

```diff
@@ -366,13 +553,13 @@ bool _isOperator(FuncRef funcRef) => [
       'uniary-',
     ].contains(funcRef.name);
 
-bool _isContructor(FuncRef funcRef, Class clazz) =>
+bool _isConstructor(FuncRef funcRef, Class clazz) =>
     funcRef.name == clazz.name || funcRef.name.startsWith('${clazz.name}.');
 
 bool _isAccessible(String member, Class clazz, DebuggerController controller) {
-  final frame = controller.selectedStackFrame.value?.frame ??
-      controller.stackFramesWithLocation.value.first.frame;
+  final frame = controller.frameForEval;
   final currentScript = frame.location.script;
-  return !member.startsWith('_') ||
-      currentScript.id == clazz?.location?.script?.id;
+  return !isPrivate(member) || currentScript.id == clazz?.location?.script?.id;
 }
+
+bool isPrivate(String member) => member.startsWith('_');
```

Comment:
This doesn't need to be public.

Reply:
actually it needs to be a public utility uses everywhere... we reinvented the wheel a lot...
lib/src/debugger/evaluate.dart:bool _isPrivate(String member) => member.startsWith('_');
lib/src/inspector/inspector_service.dart:      if (name.startsWith('_')) {
lib/src/memory/memory_graph_model.dart:                  className.startsWith('_')) ||
lib/src/memory/memory_graph_model.dart:                actual.name.startsWith('_')) ||
lib/src/provider/instance_viewer/instance_details.dart:  bool get isPrivate => name.startsWith('_');
lib/src/ui/icons.dart:    final bool isPrivate = name.startsWith('_');
lib/src/utils.dart:  final isAPrivate = a.startsWith('_');
lib/src/utils.dart:  final isBPrivate = b.startsWith('_')